app/views/departments/_form.html.erb

```diff
@@ -1,21 +1,25 @@
-<%= form_for(@department) do |f| %>
-  <% if @department.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@department.errors.count, "error") %> prohibited this department from being saved:</h2>
+<div class="new_department row">
+  <div class="col-md-6 col-md-offset-3">
+    <h1>New Department.</h1>
```

Comment:
We shouldn't put this kind of writing inside the form. The form is used for new AND edit, which is why it was set up the way it was.